src/components/cap-case.js

```diff
@@ -11,6 +11,307 @@ export default class CapCase extends LitElement {
 		case: { type: String },
 	};
 
+	static styles = [
+		css`
+			/**/
+			/* Styles for HTML under our control */
+			/**/
+
+			/* .case-container */
+
+			.case-container {
+				flex: 0 0 83.33333%;
+				max-width: 83.33333%;
+				margin: auto;
+
+				padding-top: 0;
+				padding-bottom: 100px;
+
+				/*	styles inherited from "body" in capstone */
+
+				letter-spacing: 0.01em;
+				font-size: 20px;
+				font-weight: normal;
+				line-height: 29px;
+				color: var(--color-gray-500);
+				-moz-osx-font-smoothing: grayscale;
+				-webkit-font-smoothing: antialiased;
+				-webkit-hyphens: auto;
+				-moz-hyphens: auto;
+				-ms-hyphens: auto;
+				hyphens: auto;
+			}
+
+			/* .case-header */
+
+			.case-header {
+				font-size: 0.9em;
+				font-family: "Libre Baskerville", "Baskerville", serif;
+				text-align: center;
+				padding: 2em 2em 0;
+			}
+
+			@media (min-width: 992px) {
+				.case-header {
+					padding: 0;
+				}
+
+				.case-header::after {
+					content: '*';
+					color: var(--color-gray-000);
+				}
+			}
+
+			.case-header > h1 {
+				font-family: "Libre Baskerville", "Baskerville", serif;
```

Comment:
I was initially confused that the case page defines two slightly different font stacks for serif text. But on closer look, titles set in Baskerville are quite visibly different than Source Serif, despite being both transitional typefaces. 

For consistency with the current design system, I think we could add an additional css variable to `typography.css`: 

```--font-serif-titling: "Libre Baskerville","Baskerville",serif;"```

This would more closely match how we handle two different sans-serif font stacks, with `font-sans-text` and `font-sans-titling`.  (To be most consistent, we could also rename `font-serif` to `font-serif-text`, but that's an easy one-off entirely separate PR)

Reply:
New variable added!